RenderSystems/GL3Plus/src/GLSL/src/OgreGLSLMonolithicProgram.cpp

```diff
@@ -65,7 +65,10 @@ namespace Ogre {
         {
             uint32 hash = getCombinedHash();
 
-            OGRE_CHECK_GL_ERROR(mGLProgramHandle = glCreateProgram());
+            if(!glIsProgram(mGLProgramHandle))
```

Comment:
it should be legit to just check `mGLProgramHandle == 0` here instead of calling glIsProgram

Reply:
It works indeed 👍